CriticalMass/GrowingTextView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  GrowingTextView.swift
+//
+//  Created by Kenneth Tsang on 17/2/2016.
+//  Copyright (c) 2016 Kenneth Tsang. All rights reserved.
+//
+// https://github.com/KennethTsang/GrowingTextView
```

Comment:
GrowingTextView is licensed under MIT which requires attribution in the App

Reply:
Should I add the license? Does not SPM and I just dragged it in

Reply:
Yes, it requires a license in the App. Not sure if license-plist supports multiple license sources though

Reply:
Just adding the license file could work since it is looking for those I think